lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -217,25 +235,22 @@ export default class ScheduleExceptionForm extends Component<Props> {
               <small>On these dates*</small>
             </ControlLabel>
             {ranges && ranges.map((range, index) => {
-              const startDateIndex = dates.findIndex(d => d === range.startDate)
-              const endDateIndex = dates.findIndex(d => d === range.endDate)
               return (
                 <ExceptionDateRange
                   index={index}
                   key={index}
-                  range={range}
-                  validationState={this._checkRangeValidation(startDateIndex, endDateIndex)}
+                  range={{startDate: range.startDate, endDate: range.endDate}}
```

Comment:
Why was this expansion needed?

Reply:
It wasn't actually, I modified this in https://github.com/ibi-group/datatools-ui/pull/843/commits/bcfe4911d3d622f0ebf397e12c47e6b12df3ea45